bugbug/repository.py

```diff
@@ -139,7 +139,8 @@ def download_commits(repo_dir, date_from):
     global author_experience_90_days
     for commit in commits:
         author_experience[commit] = total_commits_by_author[commit.author]
-        total_commits_by_author[commit.author] += 1
+        if not commit.ever_backedout:
+            total_commits_by_author[commit.author] += 1
```

Comment:
You'll need to also ignore it for the 90 days experience below.

The best way to ignore both would probably be to add it at the beginning of the for loop:
```Python
for commit in commits:
    if commit.ever_backedout:
        continue

    ...
```

Reply:
Adding a comment like `# We don't want to consider backed out commits when calculating author/reviewer experience`.

Reply:
Ah, actually, we will have to set author_experience and reviewer_experience anyways, otherwise we fail later on when getting from the map.